pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -10,6 +10,10 @@
 , zlib ? null, extraPackages ? [], extraBuildCommands ? ""
 , dyld ? null # TODO: should this be a setup-hook on dyld?
 , isGNU ? false, isClang ? cc.isClang or false, gnugrep ? null
+, # Prefix for binaries. Customarily ends with a dash separator.
+  prefix ? stdenv.lib.optionalString (targetPlatform != hostPlatform)
+                                     (targetPlatform.config + "-")
```

Comment:
Should this be an argument for cc-wrapper? If you want a different prefix shouldn't you change the targetPlatform?